### DISCUSSION TITLE
TSML/Mapbox won't recognize rural address - meeting won't list.

### DISCUSSION THREAD

**Discussion by AASTL** [code4recovery/12-step-meeting-list#1577] [General]
Hello,
I hope this is ok to post in this spot. I looked through other postings that seem related, but I really can't tell whether this is "same issue" with my level of knowledge.

I am working with a group in rural Missouri who want their group to show up in the Meeting Guide app.

This is the problem. The meeting rep gives me this address:

Hahn Chapel, 15574 Leghorn Lane, Marble Hill, MO 63764

When I enter that into the Twelve Step Meeting List interface, the plugin corrects the address to just this:

Hahn Chapel, Marble Hill, MO 63764

And I get an error message in TSML:   "Error: In person meetings must have a specific address."

When I go to my "import" log at the Meeting Guide dashboard, this entry shows up with "error" status.
And the meeting shows up on my website, but not in the Meeting Guide app.

Can this be corrected by hand, or some other way?

Thank you!
Tom

**Comment by anchovie91471** [comment]
@AASTL, you'll have to add a custom location. Here are the instructions for that: https://wordpress.org/plugins/12-step-meeting-list/#i%20need%20to%20correct%20a%20meeting%20address%20or%20change%20a%20pin%27s%20location

**Reply by joshreisner** [reply]
specifically, for this address you'll want to add

```php
if (function_exists('tsml_custom_addresses')) {
	tsml_custom_addresses(array(
		'15574 Leghorn Dr, Marble Hill, MO 63764, USA' => array(
			'formatted_address' => '15574 Leghorn Dr, Marble Hill, MO 63764, USA',
			'city' => 'Marble Hill',
			'latitude' => 37.306133,
			'longitude' => -89.949255,
			'approximate' => 'no',
		),
	));
}
```

**Comment by joshreisner** [comment]
hi @AASTL let's get this issue resolved. are you free today? we can do a zoom together to get this fixed

**Comment by AASTL** [comment]
Hi Josh!



I’d be very glad to get on Zoom with you today, to do this!



-Tom



*From:* Josh Reisner ***@***.***
*Sent:* Thursday, December 26, 2024 12:03 PM
*To:* code4recovery/12-step-meeting-list
*Cc:* AASTL; Mention
*Subject:* Re: [code4recovery/12-step-meeting-list] TSML/Mapbox won't
recognize rural address - meeting won't list. (Discussion #1577)



hi @AASTL <https://github.com/AASTL> let's get this issue resolved. are you
free today? we can do a zoom together to get this fixed

—
Reply to this email directly, view it on GitHub
<https://github.com/code4recovery/12-step-meeting-list/discussions/1577#discussioncomment-11670422>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/BJLYSZPT7GMFIVCGROFA6DT2HRAGZAVCNFSM6AAAAABSOY6QLKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNRXGA2DEMQ>
.
You are receiving this because you were mentioned.[image: Image removed by
sender.]Message ID:
<code4recovery/12-step-meeting-list/repo-discussions/1577/comments/
***@***.***>

**Reply by joshreisner** [reply]
great, are you free in ~15 minutes? it would be about 12:30 your time

**Comment by AASTL** [comment]
That works!



My zoom, or yours?



tom



*From:* Josh Reisner ***@***.***
*Sent:* Thursday, December 26, 2024 12:09 PM
*To:* code4recovery/12-step-meeting-list
*Cc:* AASTL; Mention
*Subject:* Re: [code4recovery/12-step-meeting-list] TSML/Mapbox won't
recognize rural address - meeting won't list. (Discussion #1577)



great, are you free in ~15 minutes? it would be about 12:30 your time

—
Reply to this email directly, view it on GitHub
<https://github.com/code4recovery/12-step-meeting-list/discussions/1577#discussioncomment-11670442>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/BJLYSZJGCCUUDOBSFTBSD3D2HRA3XAVCNFSM6AAAAABSOY6QLKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNRXGA2DIMQ>
.
You are receiving this because you were mentioned.Message ID:
<code4recovery/12-step-meeting-list/repo-discussions/1577/comments/
***@***.***>

**Reply by joshreisner** [reply]
i'll send a zoom link in a few mins

**Reply by joshreisner** [reply]
https://zoom.us/j/93805506696?pwd=UXKevzTaJUMHaL4oYSAIW9z4kbve8i.1
